curriculum/challenges/english/blocks/lab-pricing-plans-layout/68e45a2b4ab9b2f48b1cc6df.md

```diff
@@ -0,0 +1,414 @@
+4. Your `basic-plan` element should have an `h2` element with the text `Basic Plan`.
+5. Your `basic-plan` element should have a `p` element with the text `$9/month`.
+6. Your `pro-plan` element should have an `h2` element with the text `Pro Plan`.
+7. Your `pro-plan` element should have a `p` element with the text `$19/month`.
+8. Your `premium-plan` element should have an `h2` element with the text `Premium Plan`.
+9. Your `premium-plan` element should have a `p` element with the text `$29/month`.
+10. Each of your `.pricing-card` elements should use Flexbox to stack its content in a column and justify the space between the children using `space-between`. Each element should also set the `flex` property to `0 0 200px` to give it a consistent width and prevent it from growing or shrinking in the layout. Each element should set the `border` property to `2px solid black` to see how the different cards take up space.
+11. The `basic-plan` element should appear first in the layout. You should use the `order` property for this.
+12. The `pro-plan` element should appear second in the layout. You should use the `order` property and set its `flex-grow` property to `2` so it takes up more space than the other plans.
+13. The `premium-plan` element should come last in the layout. You should use the `order` property for this.
+
+**Note:** Be sure to link your stylesheet in your HTML and apply your CSS.
+
+# --hints--
+
+Your `h1` element should contain the text `Pricing Plans`.
+
+```js
+const h1 = document.querySelector('h1');
+assert.strictEqual(h1?.textContent, 'Pricing Plans');
+```
+
+Your `div` element should contain the class `pricing-container`.
+
+```js
+const div = document.querySelector('.pricing-container');
+assert.strictEqual(div?.tagName, 'DIV');
+```
+
+Your `pricing-container` element should contain the property `display` and the value `flex`.
+
+```js
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.pricing-container')?.display, 'flex');
+```
+
+Your `pricing-container` element should contain the property `flex-wrap` and the value `wrap`.
+
+```js
+assert.strictEqual(new __helpers.CSSHelp(document).getStyle('.pricing-container')?.flexWrap, 'wrap');
+```
+
+You should have three `div` elements with the class `pricing-card`.
+
+```js
+const divEls = document.querySelectorAll('.pricing-card');
+assert.lengthOf(divEls, 3);
+```
+
+Your `div` element should contain the class `pricing-card` and the class `basic-plan`.
+
+```js
+const div = document.querySelector('.pricing-card.basic-plan');
+
+assert.strictEqual(div?.tagName, 'DIV');
+assert.isTrue(div?.classList.contains('basic-plan'));
+assert.isTrue(div?.classList.contains('pricing-card'));
+```
+
+Your `div` element should contain the class `pricing-card` and the class `premium-plan`.
+
+```js
+const div = document.querySelector('.pricing-card.premium-plan');
+
+assert.strictEqual(div?.tagName, 'DIV');
+assert.isTrue(div?.classList.contains('premium-plan'));
```

Comment:
this is redundant, you are selecting the element with the class, so this and the line below will be true by default

you may want to get the element using a `div:nth-of-type()` selector for all three tests like this